### ISSUE TITLE
Incorrect resolution of a plugin name

### ISSUE THREAD

**Issue by moltar** [eslint/eslint#11491]
**Tell us about your environment**

* **ESLint Version:** v5.3.0
* **Node Version:** v11.6.0
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "parserOptions": {},
  "rules": {
    "prettier/prettier": [
      "error",
      {
        "singleQuote": true,
        "semi": false,
        "trailingComma": "all",
        "printWidth": 100,
        "usePrettierrc": false
      }
    ]
  },
  "env": {
    "browser": true,
    "commonjs": true,
    "es6": true,
    "jest": true,
    "node": true
  },
  "extends": [
    "plugin:notice"
  ]
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Include a plugin without a slash.

Error is produced (notice the name is missing the last letter):

```
Error: Cannot find module 'eslint-plugin-notic'
```

**What did you expect to happen?**

To work.

**What actually happened? Please include the actual, raw output from ESLint.**

Threw.

**Are you willing to submit a pull request to fix this bug?**

Maybe.

**Comment by moltar** [comment]
The bug is here:

https://github.com/eslint/eslint/blob/6693161978a83e0730d5ea0fecdb627c5a2acdfd/lib/config/config-file.js#L491

**Comment by g-plane** [comment]
Hey @moltar , thanks for this issue.

Are you using a plugin? If so, have you added it to your ESLint config file to `plugins` field?

**Comment by moltar** [comment]
Workaround is to use:

```
"plugin:notice/"
```

**Comment by moltar** [comment]
@g-plane I added it as part of `extends` option.

**Comment by g-plane** [comment]
You need to add something like this:

```json
{
  "plugins":["notice"]
}
```

**Comment by moltar** [comment]
Okay, I thought for some reason that `"plugin:notice/"` does the same thing. At least it does for some other plugins, like pretty and standard.

**Comment by moltar** [comment]
Thanks!
